shimmingtoolbox/cli/create_coil_profiles.py

```diff
@@ -291,6 +296,147 @@ def create_coil_profiles_cli(fname_json, path_relative, autoscale, unwrapper, th
     logger.info(f"Filename of the coil config file is: {fname_coil_config}")
 
 
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+)
+@click.option('-i', '--input', 'fname_txt', type=click.Path(exists=True), required=True,
+              help="Input filename of wires' geometry text file. "
+                   "See the `tutorial <https://shimming-toolbox.org/en/latest/user_section/tutorials.html>`_ for more "
+                   "details.")
+@click.option('--fmap', 'fname_fmap', required=True, type=click.Path(exists=True),
+              help="Static \u0394B\u2080 fieldmap on which to calculate coil profiles. Only FOV and affine are used.")
+@click.option('--offset', 'offset', required=False, type=(float, float, float), default=(0,0,0),
+              help="XYZ offset: The difference between the coil’s isocenter position and the field map's isocenter position (in mm). \n Defaulted to (0,0,0).")
+@click.option('--flip', 'dims_to_flip', required=False, type=(float, float, float), default=(1,1,1),
+              help="Dimensions (XYZ order) to flip in the wires' geometry (1 for no flip, -1 for flip). Defaulted to 1 1 1.")
+@click.option('--software', type=click.Choice(['AutoCAD']), default='AutoCAD',
+              help=f"Software from which the geometries were extracted.")
+@click.option('--coil_name', 'coil_name', required=False, type=click.STRING, default="new",
+              help="Name of the coil. If not provided, \"new\" will be used.")
+@click.option('--min', 'min_current', required=False, type=click.FLOAT, default=-1,
+              help="The minimum current in amps going through each channel. Defaulted to -1 A.")
+@click.option('--max', 'max_current', required=False, type=click.FLOAT, default=1,
+              help="The maximum current in amps going through each channel. Defaulted to 1 A.")
+@click.option('--max_sum', 'max_current_sum', required=False, type=click.FLOAT, default=None,
+              help="The maximum sum of currents in amps going through all loops. Defaulted to the number of channel.")
+@click.option('-o', '--output', 'fname_output', type=click.Path(), required=False,
+              default=os.path.join(os.path.curdir, 'coil_profiles.nii.gz'),
+              help="Output filename of the coil profiles NIfTI file. Supported types : '.nii', '.nii.gz'")
+@click.option('-v', '--verbose', type=click.Choice(['info', 'debug']), default='info', help="Be more verbose")
+def from_cad(fname_txt, fname_fmap, offset, dims_to_flip, software, coil_name, min_current, max_current, max_current_sum, fname_output, verbose):
+    """ Testing change Create \u0394B\u2080 coil profiles from CAD wire geometries."""
+    # Assert inputs
+    assert min_current < max_current, f"Minimum current should be smaller than maximum current ({min_current} >= {max_current})"
```

Comment:
Raise errors if a condition is not respected. Use `assert` in tests or when debugging.